### ISSUE TITLE
Mediaroom device status

### ISSUE THREAD

**Issue by moonlightz** [dgomes/pymediaroom#11]
Hi there, I own a meobox (I am trying to recall what make is) which runs mediaroom windows ce 5. The story is kinda complicated but resuming it is that I'm "coding" an irc bot made of tcl running on openwrt which runs on a router and I recently made a simple tcl for the bot which controls the mediaroom device. It was a bit of trial-error.
So far I can command like "meobox 45" (from irc channel) to go to the channel 45 and the script sees 45 and sends key=52\nkey=53\n to the port 8082, connecting to 192.168.1.64 first it isn't. I can also send like "meobox p+" for channel up and so on. Also "meobox key=number" for a custom key which I used to map keys. For the first trys, I only mapped keys upto 60 back and forward because some keys will only work if menus are on-screen.
I ended up having to dig the Internet to get the other keys mapped like power volume+ .... The script is trivial and just does what is meant for. Yet to map letters now that I think for but I will have to do that for the following days. It can also recall commands like "meobox adic num button-name" or "meobox rem num" to remove from the list.

Sample
```
[05:02:01] <@moonlight2> meobox power
[05:02:00] <%TP-Link> ok     <<- responde from telnet
[06:13:53] <@moonlight2> meobox listar
[06:13:53] <%TP-Link> Lista de botões atribuidos:
[06:13:53] <%TP-Link> 8 esc2         33 p+          38 cima        112 guiatv     117 prev       123 stop       143 blue       174 v-       
[06:13:53] <%TP-Link> 13 ok          34 p-          39 >           114 vclube     118 rewind     140 red        156 swscreen   175 v+       
[06:13:53] <%TP-Link> 19 pausa       36 menu        40 baixo       115 gravacoes  121 forward    141 green      159 i          225 record   
[06:13:53] <%TP-Link> 27 esc         37 <           111 opcoes     116 teletexto  122 next       142 yellow     173 mute       233 power    
[06:14:57] <@moonlight2> meobox 45
[06:14:57] <%TP-Link> ok
```

Anyways, I'm not here to advertise my bot, scripts... Moving on...
Maybe 1 or 2 or maybe more years ago, I came across with ....dvrfs/info.xml... and how to get the info out of the meobox and never seem to find any useful information and gave up.
I wonder if there is a way to get the status of the mediaroom like if it is power off or if it is powered on and what channel is on the tv and what volume from 1-25 like the app "meo remote" from android's play store does.
I tried wget 192.168.1.64:8080/dvrfs/info.xml and got a http error 400 bad request. I don't really know what I am doing. The meobox has 8080 8082 8086 ports opened.
I found your pymediaroom by chance and run python3 app.py and it seems really interesting.
I wanted to know what you think regarding getting the status of the mediaroom. If is there a plan to build a python script to get the status. The app.py does seem to give us debug.
Regards

**Comment by dgomes** [comment]
Olá :)

Nenhum operador está instalar Mediaroom hoje em dia, e resultado disso este projecto está em modo "maintenance only" 

Nunca consegui perceber toda a info que a box "disponibiliza", os canais por exemplo são grupos IP multicast diferentes, mas mudam de região para região... 

Boa sorte com o teu projecto!

**Comment by moonlightz** [comment]
Obrigado na mesma. Na boa. :-)
Como se costuma dizer... "if there is a will, there is a way"... Irei provavelmente conseguir ou não. Pelo que vi, alguns ainda tentaram mas não conseguiram. Provavelmente tenho de pôr qualquer coisa a ouvir o tráfego entre a app e a meobox, e depois tentar decifrar o que eles dizem entre eles. Meter um "meobox info" ou "meobox estado" seria a cereja no topo do bolo.

**Comment by FragMenthor** [comment]
> Obrigado na mesma. Na boa. :-) Como se costuma dizer... "if there is a will, there is a way"... Irei provavelmente conseguir ou não. Pelo que vi, alguns ainda tentaram mas não conseguiram. Provavelmente tenho de pôr qualquer coisa a ouvir o tráfego entre a app e a meobox, e depois tentar decifrar o que eles dizem entre eles. Meter um "meobox info" ou "meobox estado" seria a cereja no topo do bolo.

Algum progresso nisto? É das coisas que mais me frustra desde que iniciei no HA...

**Comment by moonlightz** [comment]
> > Obrigado na mesma. Na boa. :-) Como se costuma dizer... "if there is a will, there is a way"... Irei provavelmente conseguir ou não. Pelo que vi, alguns ainda tentaram mas não conseguiram. Provavelmente tenho de pôr qualquer coisa a ouvir o tráfego entre a app e a meobox, e depois tentar decifrar o que eles dizem entre eles. Meter um "meobox info" ou "meobox estado" seria a cereja no topo do bolo.
> 
> Algum progresso nisto? É das coisas que mais me frustra desde que iniciei no HA...

Não, quase nada. Eu usei o Bluestacks+app Meo Remote e há algum tráfego mas aonde quer que ponha o Nirsoft Smartsniff, vejo que há actividade na minha rede interna como se tivesse a divulgar/anunciar. Tentei ver se fazia algum sentido alguma coisa e até faz mas falta troca de cookies e auths e outras porcarias. 
Como se o azar não bastasse, a MEO entretanto descartou um dos seus canais mais importantes e tenho estado ocupado a continuar a extrair episódios das gravações feitas do cfanal e com outras coisas.
